tests/eas/fork_migration.config

```diff
@@ -0,0 +1,53 @@
+{
+    "modules"  : [ "bl" ],
+    "exclude_modules"  : [ "hwmon" ],
+    "tools"    : [ "rt-app" ],
+    "ftrace" : {
+        "events" : [
+            "sched_energy_diff",
+            "sched_load_avg_task",
+            "sched_load_avg_cpu",
+            "sched_migrate_task",
+            "sched_switch"
+        ],
+    },
+    "wloads" : {
+        // Create N 100% tasks and M 10% tasks which run in parallel, where N is
+        // the number of big CPUs and M is the number of LITTLE CPUs.
+        "fmig" : {
+            "type" : "rt-app",
+            "conf" : {
+                "class" : "profile",
+                "params" : {
+                    "small" : {
+                        "kind" : "Periodic",
+                        "params" : {
+                            "duty_cycle_pct": 10,
```

Comment:
Note that these values used to be configured by values in acceptance.config, but those values can no longer be used here. I think this can be something that we discuss later along with workload description in general, along with what @derkling mentioned [previously](https://github.com/ARM-software/lisa/pull/197#discussion_r80811949)